test_requirements/django-2.2.txt

```diff
@@ -1,4 +1,4 @@
 -r requirements_base.txt
-Django>=2.2,<3.0
```

Comment:
@marksweb Do we drop 2.2?

Reply:
@vinitkumar certainly needs to happen at some point. I think we could consider ourselves in a transition. Maybe once the v4 branches to support various Django versions are sorted we can talk about dropping 2.2.